examples_tests/39.DenoiserTonemapper/main.cpp

```diff
@@ -65,7 +68,7 @@ int main(int argc, char* argv[])
 	params.Vsync = true;
 	params.Doublebuffer = true;
 	params.Stencilbuffer = false;
-	params.StreamingDownloadBufferSize = 256*1024*1024; // change in Vulkan fo
+	params.StreamingDownloadBufferSize = 2560*1024*1024;
```

Comment:
after generating the DDSes is there any need for such a ridiculously huge download buffer?

Reply:
no need actually

Reply:
unfortunately we need more space though

Reply:
what for?